authors/apps/authentication/test_authentication.py

```diff
@@ -0,0 +1,48 @@
+import json
+from django.urls import reverse
+from rest_framework.views import status
+from rest_framework.test import APITestCase, APIClient
+from .models import UserManager
+
+
+class TestUsers(APITestCase):
+    client = APIClient()
```

Comment:
hey, Henry.. it could be better if you defined the test client in a setup method. 
class TestUsers(APITestCase):
    def setUp(self):
        client = APIClient()

Reply:
Hey, Henry, the second commit message is not like we had agreed.

Reply:
alright Josh
I will do better next time, sometimes I forget and go back to default settings

Reply:
haha ok